src/components/SpecialResolution/Memorandum.vue

```diff
@@ -0,0 +1,440 @@
+<template>
+  <v-card
+    id="memorandum"
+    flat
+  >
+    <!-- Header -->
+    <div class="section-container header-container">
+      <v-icon color="appDkBlue">
+        mdi-format-list-text
+      </v-icon>
+      <label class="font-weight-bold pl-2">Memorandum</label>
+    </div>
+
+    <v-form ref="memorandumForm">
+      <div
+        id="memorandum-section"
+        class="section-container"
+        :class="{'invalid-section': memorandumEditingInvalid}"
+      >
+        <v-row
+          no-gutters
+          class="mt-4"
+        >
+          <v-col
+            cols="2"
```

Comment:
Does this line up with the other sections on this page? (In other filings, this is cols="3".)

Reply:
OK, I see it depends if you are in edit or review mode. Fixed, thanks.